src/Compilers/VisualBasic/Portable/Symbols/ReferenceManager.vb

```diff
@@ -361,9 +359,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
 
                 ' Setup references for the compilation
-                referencedAssembliesMap = New Dictionary(Of MetadataReference, ReferencedAssembly)(referenceMap.Length)
+                referencedAssembliesMap = New Dictionary(Of MetadataReference, Integer)(referenceMap.Length)
                 referencedModulesMap = New Dictionary(Of MetadataReference, Integer)(modules.Length)
 
+                Dim referencedAssemblies = ArrayBuilder(Of AssemblySymbol).GetInstance()
+                Dim aliasesOfReferencedAssemblies = ArrayBuilder(Of ImmutableArray(Of String)).GetInstance()
+
                 Dim sourceModule = assemblySymbol.SourceModule
                 Dim referencedAssemblySymbols = sourceModule.GetReferencedAssemblySymbols()
```

Comment:
That's a good question, I'm not sure. Will look into whether we can reuse the existing array.